roles.json

```
@@ -131,9 +131,9 @@
         "responsibilities": {
             "description": "Plan, oversee, and disburse the Astropy project finances, including:",
             "details": [
-                "Determine the process for paying people from Astropy Project-level funding.",
+                "Determine and manage the process for paying people from Astropy project-level funding.",
```

When you change "project" to lowercase, the meaning is a bit different? I thought "Astropy Project" is a proper noun?
```suggestion
                "Determine and manage the process for paying people from Astropy's project-level funding.",
```

---

I don't disagree with you, but I was aiming for consistency with the rest of the page — it starts “The Astropy project...”!

I like your wording though; it neatly sidesteps the issue (in this text, at least). 👍